src/pages/Auth/auth.slice.js

```diff
@@ -22,6 +27,14 @@ const handleAuthFulfilled = (state, action) => {
   localStorage.setItem(LocalStorage.accessToken, access_token)
 }
 
+const handleUpdateProfileFulfilled = (state, action) => {
+  state.profile = action.payload.data
+  localStorage.setItem(
```

Comment:
ngoài cách này ra, em còn có thể save localStorage bằng cách này nữa nhé, ý tưởng cơ bản là bất cứ khi nào em change state thì nó sẽ update lại localStorage cho em luôn. Ví dụ như khi em add hoặc remove 1 cái gì đó, thay vì em update localStorage 2 lần ở 2 action đó thì chỉ còn gói gọn lại thành 1 mà thôi
https://stackoverflow.com/questions/67248867/react-redux-toolkit-how-can-i-can-store-my-initialstate-to-localstorage-using-c

Reply:
Vâng để em tìm hiểu thêm